tests/app_settings.rs

```diff
@@ -944,7 +944,7 @@ fn aaos_opts_mult() {
         .setting(AppSettings::AllArgsOverrideSelf)
         .arg(
             Arg::from("--opt [val]... 'some option'")
-                .number_of_values(1)
+                .grouped_min_values(1)
```

Comment:
When args is multiple, AllArgsOverrideSelf is disabled. So here we originally use number_of_values is actually bad.